deCODE_to_MT.py

```diff
@@ -90,8 +89,17 @@ def main(
         )
         | (hl.len(mt.alleles) != 2)
         | ((hl.len(mt.alleles) == 2) & (mt.n_unsplit_alleles != 2))
-        | (hl.is_missing(mt["InbreedingCoeff"]))
-        | ((hl.is_defined(mt["InbreedingCoeff"])) & (mt["InbreedingCoeff"] < -0.3))
+    )
+    mt = mt.filter_rows(filter_conditions, keep=False)
+
+    # Exclude variants with inbreeding coefficient < -0.3
+    # InbreedingCoeff was calculated by function 'bi_allelic_site_inbreeding_expr'
+    mt = hl.variant_qc(mt)
+    mt = mt.annotate_rows(InbreedingCoeff=bi_allelic_site_inbreeding_expr(mt.GT))
+
+    # Filter variants with InbreedingCoeff
+    filter_conditions = (hl.is_missing(mt["InbreedingCoeff"])) | (
```

Comment:
It should never be missing, right? Even if it were, you can simplify this condition to `hl.is_missing(mt["InbreedingCoeff"]) | (mt["InbreedingCoeff"] < -0.3))`.

Reply:
Thanks for spotting it! I did find `None` in test dataset, but I guess I should use `hl.is_defined()` to filter them...I have now simplied this filter as `mt = mt.filter_rows(mt.InbreedingCoeff >= -0.3, keep=True)`.